larndsim/config/config.yaml

```diff
@@ -62,12 +62,12 @@ module0:
     SIM_PROPERTIES:  singles_sim_mod2mod_variation.yaml
 
 # nd-lar (same module configuration)
-# NuMI beam (not realistic)
```

Comment:
Is there something baked into `larnd-sim` related to the beam or if the LBNF flux was used further upstream is the `larnd-sim` output then "realistic" for `NDLAr`?

Reply:
I think at the time of writing the configuration, LBNF flux was not tried upstream to `larnd-sim`. Your edits makes sense